modules/nw-networkpolicy-about.adoc

```diff
@@ -68,11 +68,18 @@ spec:
   policyTypes:
   - Ingress
 ----
+
++
+If the Ingress Controller is configured with `endpointPublishingStrategy: HostNetwork`, then the Ingress Controller Pod runs on the host network.
+When running on the host network, the traffic from the Ingress Controller is assigned `netid:0`.
+The `netid` for the namespace associated with the Ingress Operator is different, so the `matchLabel` in the `allow-from-openshift-ingress` network policy will not work as expected.
```

Comment:
s/associated/that is associated

s/will not/does not

At this point, will the user know what is expected?